precise_driver/config/precise_controllers.yaml

```diff
@@ -26,6 +26,8 @@ hardware_interface:
   in_range: -1
   straight: 0 #0=rotational; -1=straight
 
+  post_init_configuration: [0.085, -1.552, 3.136, 3.155, 0.038] #config where the arm should move to after successful initialization
```

Comment:
how about a mapping of joints and values...then you could also only move selected joints and leave the others where they are...

Reply:
ok, could be an additional feature. For know I think it's ok to give a complete arm configuration. Then we know that the arm has a valid configuration after init. If we find a need for that feature in future we can add it...